### DISCUSSION TITLE
Probe-array and PID

### DISCUSSION THREAD

**Discussion by bravowren** [artisan-roaster-scope/artisan#1212] [Q&A]
I have been using a probe array for BT as outlined by Marko in the Oct 2021 blog, by specifying BT as  (B3+B5)/2 under config>devices>Symb ET/BT.

I am about to add PID control using Artisan software PID. I watched with great interest Jason Scott's video on the features of Artisan 2.8.2 in Dec 2022. He was specially excited about the new feature of  enabling the selection any device as the PID source (previously was just ET or BT). In this video he set up a new device for the probe array using the same calc.

My question: is there any advantage to set up a virtual device ala Jason per the enclosed image vs. just using BT as the source (with my symbolic assignment)? 
![image1](https://github.com/artisan-roaster-scope/artisan/assets/81058869/ebbe7759-2e50-48cd-ae33-5e32c84a3591)

**Comment by MAKOMO** [comment]
There should be no difference
